mmrazor/models/mutables/mutable_channel/slimmable_mutable_channel.py

```diff
@@ -83,7 +83,9 @@ def num_choices(self) -> int:
 
     def convert_choice_to_mask(self, choice: int) -> torch.Tensor:
```

Comment:
suggestion: let MutableChannel only deal with mask, and implement the process which converts int/float/choice to mask, in another Mutable